docs/source/deprecation.rst

```diff
@@ -0,0 +1,19 @@
+FiftyOne Deprecation Notices
+============================
+
+.. default-role:: code
+
+Python 3.8
+----------
+*Support Ends October 2024*
+
+`Python 3.8 <https://devguide.python.org/versions/>`_
+transitions to `end-of-life` effective October of 2024. FiftyOne releases after
+September 30, 2024 will no longer support Python 3.8.
+
+Versions of `fiftyone` after 0.24.1, or after `fiftyone` Teams version 0.18.0,
```

Comment:
nit: suggest "FiftyOne Teams 0.18.0" instead of "`fiftyone` Teams version 0.18.0", since Teams versions refer to the product, not the package.